src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml

```diff
@@ -53,3 +53,9 @@ services:
           'es': 'http://doc.prestashop.com/display/PS17/Gestionar+Atributos+del+Producto'
           'it': 'http://doc.prestashop.com/display/PS17/Gestire+gli+Attributi+dei+Prodotti'
           '_fallback': 'http://doc.prestashop.com/display/PS17/Managing+Product+Attributes'
+        'monitoring':
+          'en': 'http://doc.prestashop.com/display/PS17/Monitoring+your+Catalog'
```

Comment:
Let's use https instead. I see previous links were using http but this is a mistake, I'll fix it.

Security matters 🍾

Reply:
We may have to change the documentation too then? https://docs.google.com/document/d/1xeiEwT02OpBxePesgdd68HxKD6XZe6PqYAT3KD76ol4